helm/private-network-ibft/besu/README.md

```diff
@@ -43,7 +43,7 @@ node:
 
 #### 4. Run helm and install the chart
 ```bash
-helm install --namespace NAMESPACE --name pantheon ./pantheon
+helm install --namespace NAMESPACE --name besu ./besu
 ```
 
 #### 5. In the dashboard, you will see each bootnode deployment & service, nodes & a node service, miner if enabled, secrets(opaque) and a configmap
```

Comment:
Line 61 there is a `PANTHEON_NODE_SERVICE_HOST` that was missed

Reply:
fixed